rel/files/ejabberd.cfg

```diff
@@ -662,18 +662,15 @@
                            % default: 600 seconds
                            % {resume_timeout, 600}
                           ]},
-%  {mod_muc_light, [
-%                   {host, "muclight.@HOST@"}
-%                  ]},
-  {mod_muc, [
```

Comment:
Did XSF obsolete MUC? We can say it's obsolete, I'm fine with it but then we should enable by default `mod_muc_light`.